activemodel/lib/active_model/validations/inclusion.rb

```diff
@@ -33,6 +34,8 @@ module HelperMethods
       #   with <tt>Range#cover?</tt>, otherwise with <tt>include?</tt>. When using
       #   a proc or lambda the instance under validation is passed as an argument.
       # * <tt>:within</tt> - A synonym(or alias) for <tt>:in</tt>
+      # * <tt>:all</tt> - If attribute is an <tt>Array</tt> validates the inclusion of
```

Comment:
I didn't document this option for `validates_exclusion_of` (even if is available in the code path) because I couldn't find a valid use case.